backend/src/test/java/com/example/backend/service/TripServiceTest.java

```diff
@@ -43,28 +44,28 @@ void add_addsTripToRepoAndReturnsTrip() {
         // given
         Trip trip = new Trip("abc1", "My Trip", List.of(
```

Comment:
Though this is probably legitimate, you are testing 2 distinct features simultaneously: The general add functionality, and the automatic insertion of a location id if none is given. Would there be a point in separating the two into two individual tests?

Reply:
Guter Punkt. Du Meinst dass ich einmal einen Test habe, der nur das "Trip add" testet und dann einen Test, der prüft, dass die `add`Methode auch wirkjlich location IDs generiert?
Bau ich um 👍

Reply:
done